### PR TITLE
added get variation strings

### PR DESCRIPTION
**Pull request by beliaev-maksim**

### CODE REVIEW

File: pyaedt/application/Analysis.py [hunk 1]
```diff
@@ -480,6 +480,28 @@ def variations(self, setup_sweep=None):
...
+            -------
+            list of str
+                List of variation families.
+
+            """
+            if not setup_sweep:
+                setup_sweep = self._app.existing_analysis_sweeps[0]
+            vs = self._app.osolution.GetAvailableVariations(setup_sweep)
+
+            return vs
```
Comment (akaszynski) [inline pyaedt/application/Analysis.py:503]:
```suggestion
            return self._app.osolution.GetAvailableVariations(setup_sweep)
```

File: pyaedt/application/Analysis.py [hunk 2]
```diff
@@ -480,6 +480,28 @@ def variations(self, setup_sweep=None):
                 families.append(family)
             return families
 
+        @aedt_exception_handler
+        def get_variation_strings(self, setup_sweep=None):
+            """
+            Variations.
```
Comment (akaszynski) [inline pyaedt/application/Analysis.py:486]:
```suggestion
            """Return variation strings.
```

File: pyaedt/application/Analysis.py (hunk 1 above)
Comment (beliaev-maksim) [reply pyaedt/application/Analysis.py]: @akaszynski 
thanks!

**Review by maxcapodi78 — APPROVED** [review]
LGTM. I'll add UT to it

**Comment by beliaev-maksim** [comment]
@maxcapodi78 
thanks!

**Pull request merged**